Applications/Spire/Source/Ui/OrderStatusComboBox.cpp

```diff
@@ -0,0 +1,34 @@
+#include "Spire/Ui/OrderStatusComboBox.hpp"
+#include <QHBoxLayout>
+#include "Beam/Collections/EnumIterator.hpp"
```

Comment:
Treat Beam as a third party library, use ```<...>``` to include it.

Reply:
So alphabetically it now is included before <QHBoxLayout>?

Reply:
Included before what?

Reply:
Mean to say that would `<QHBoxLayout>` be included after Beam includes?